project/Bootstrap.scala

```diff
@@ -0,0 +1,34 @@
+package sbt
```

Comment:
I still don't understand why this is needed.

Reply:
It is explained in the comments below

Reply:
```scala
// `extendedClassLoaderCache` is package private in package sbt
val cache = state.extendedClassLoaderCache
```

Reply:
https://github.com/sbt/sbt/blob/40870d3fa84b005f70042948d81d21b3aae2d820/main-command/src/main/scala/sbt/State.scala#L399

Reply:
I agree now that it would be easier if `makeScalaInstance` is public in sbt...

Reply:
Ah I missed that, so can we make this method public in sbt? It seems like the only way to properly use ScalaInstance which is public. Otherwise our build could break at any update of sbt.

Reply:
https://github.com/sbt/sbt/pull/6480